RecoCTPPS/Configuration/python/RecoCTPPS_EventContent_cff.py

```diff
@@ -0,0 +1,39 @@
+import FWCore.ParameterSet.Config as cms
+
+RecoCTPPSFEVT = cms.PSet(
+  outputCommands = cms.untracked.vstring(
+    'keep TotemFEDInfos_totemRPRawToDigi_*_*',
+    'keep TotemTriggerCounters_totemTriggerRawToDigi_*_*',
+    'keep TotemRPDigiedmDetSetVector_totemRPRawToDigi_*_*',
+    'keep TotemVFATStatusedmDetSetVector_totemRPRawToDigi_*_*',
+    'keep TotemRPClusteredmDetSetVector_totemRPClusterProducer_*_*',
+    'keep TotemRPRecHitedmDetSetVector_totemRPRecHitProducer_*_*',
+    'keep TotemRPUVPatternedmDetSetVector_totemRPUVPatternFinder_*_*',
+    'keep TotemRPLocalTrackedmDetSetVector_totemRPLocalTrackFitter_*_*'
+  )
+)
+
+
+RecoCTPPSRECO = cms.PSet(
+  outputCommands = cms.untracked.vstring(
+    'keep TotemFEDInfos_totemRPRawToDigi_*_*',
+    'keep TotemTriggerCounters_totemTriggerRawToDigi_*_*',
+    'keep TotemRPDigiedmDetSetVector_totemRPRawToDigi_*_*',
+    'keep TotemVFATStatusedmDetSetVector_totemRPRawToDigi_*_*',
+    'keep TotemRPClusteredmDetSetVector_totemRPClusterProducer_*_*',
+    'keep TotemRPRecHitedmDetSetVector_totemRPRecHitProducer_*_*',
+    'keep TotemRPUVPatternedmDetSetVector_totemRPUVPatternFinder_*_*',
+    'keep TotemRPLocalTrackedmDetSetVector_totemRPLocalTrackFitter_*_*'
+  )
+)
+
+
+RecoCTPPSAOD = cms.PSet(
+  outputCommands = cms.untracked.vstring(
+    'keep TotemVFATStatusedmDetSetVector_totemRPRawToDigi_*_*',
+    'keep TotemRPClusteredmDetSetVector_totemRPClusterProducer_*_*',
+    'keep TotemRPRecHitedmDetSetVector_totemRPRecHitProducer_*_*',
+    'keep TotemRPUVPatternedmDetSetVector_totemRPUVPatternFinder_*_*',
+    'keep TotemRPLocalTrackedmDetSetVector_totemRPLocalTrackFitter_*_*'
```

Comment:
After looking at the outputs now with #14605 running, I realize that AOD does not have digis (TotemRPDigiedmDetSetVector).
Considering that you may need to rerun your algorithms, and that AOD is the most easily accessible data tier,
I suggest to add digis to AOD (can add them in #14605).